test/test_other.py

```diff
@@ -13194,7 +13194,7 @@ def test_itimer_pthread(self):
     self.do_other_test('test_itimer.c')
 
   def test_itimer_standlone(self):
-    self.emcc_args += ['-sSTANDALONE_WASM']
+    self.emcc_args += ['-sSTANDALONE_WASM', '-sWASM_BIGINT']
```

Comment:
Why did this change?

Reply:
`-sSTANDALONE_WASM` without `-sWASM_BIGINT` has always generated runtime warnings: 

https://github.com/emscripten-core/emscripten/blob/68ae550a0a6d64454e306c27fa72abbf173de773/src/postamble.js#L415

After this change we now get an actual runtime error related to this due to the second argument to clock_time_get:

```
TypeError: Cannot mix BigInt and other types, use explicit conversions
    at convertI32PairToI53Checked (/usr/local/google/home/sbc/dev/wasm/emscripten/out/test/test_itimer_standalone.js:1192:24)
    at _clock_time_get (/usr/local/google/home/sbc/dev/wasm/emscripten/out/test/test_itimer_standalone.js:1197:29)
    at __clock_gettime (wasm://wasm/0007fe0e:wasm-function[12]:0x57b)
    at emscripten_get_now (wasm://wasm/0007fe0e:wasm-function[15]:0x607)
    at setitimer (wasm://wasm/0007fe0e:wasm-function[22]:0x7ea)
    at __original_main (wasm://wasm/0007fe0e:wasm-function[6]:0x3eb)
    at _start (wasm://wasm/0007fe0e:wasm-function[7]:0x4c9)
    at /usr/local/google/home/sbc/dev/wasm/emscripten/out/test/test_iti
```

Before this was silently ignored.. but I think failing in this way when calling an i64-taking function in standalone more without WASM_BIGINT is the expected behaviour.